frontend/.env.local

```
@@ -0,0 +1,3 @@
+CARD_VALIDATION_API_URL=http://localhost:5000/api/cards/validate
+CARD_VALIDATION_TOKEN=8234d078-e3ab-479e-a20e-89eb4dd0133f
+CARD_VALIDATION_CLIENT=3W6izon01E77goCGve8pHA
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)